packages/vue/src/stories/releases/v0.12.x/v0.12.0/v0.12.0.stories.mdx

```diff
@@ -20,6 +19,7 @@ v0.12.0 contains API breaking changes and new features.
 ## 🐛 Fixes
 
 - SfTabs: fixed issue with memory leak
-- transitions: remove SfExpand functional component and fix sf-expand css styles 
+- transitions: remove SfExpand functional component and fix sf-expand css styles
+- fix: required attribute added to the link prop in SfLink component
```

Comment:
what do you think about changing this description a little bit and moving it to Breaking Changes? It'd be more readable if the first one will be a component where've been made changes, like this: `- SfLink: /descriprion of changes/`

Reply:
Sure will update the changelog to make it more descriptive.